Gemfile.lock

```diff
@@ -55,16 +55,17 @@ GEM
     ffi (1.15.5)
     figaro (1.2.0)
       thor (>= 0.14.0, < 2)
-    haml (5.2.2)
-      temple (>= 0.8.0)
+    haml (6.3.0)
+      temple (>= 0.8.2)
+      thor
       tilt
     haml-lint (0.999.999)
```

Comment:
what...?

Reply:
Interesting. That's apparently a dummy gem that does nothing but a) depend on `haml_lint` and b) attract your attention with its weird version number.

It's to prevent typo-squatting when someone ("someone") adds the `haml-lint` to their `Gemfile` instead of the correct `haml_lint`.